### PR TITLE
Added attributes (rebase)

### PR DESCRIPTION
**Pull request by RomanSek**
Cleaned up changes from https://github.com/PiwikPRO/PPMS-PublicDocs/pull/71 PR.
Includes rebase and some conflicts resolution so there may be errors.

### CODE REVIEW

File: platform/authorized_api/apps/index.json [hunk 1]
```diff
@@ -10,6 +10,11 @@
     }
   ],
   "definitions": {
+    "App_Id": {
```
Comment (RomanSek) [inline platform/authorized_api/apps/index.json]: Looks like it's not used anywhere. I'm not sure if this is unnecessary or if I made an error during conflict resolution.

File: platform/authorized_api/apps/index.json [hunk 2]
```diff
@@ -329,8 +356,8 @@
           "example": true
         },
         "gdprUserModeEnabled": {
-          "description": "Gdpr User Mode enabled or disabled for an app",
           "type": "boolean",
+          "description": "Gdpr User Mode enabled or disabled for an app",
```
Comment (RomanSek) [inline platform/authorized_api/apps/index.json]: Looks like moving descriptions in these params was not introduced in backport fixes.

File: platform/authorized_api/apps/index.json (hunk 1 above)
Comment (chriskicker) [reply platform/authorized_api/apps/index.json]: yep, `App_Id` and `App` are unused

File: platform/authorized_api/apps/index.json (hunk 2 above)
Comment (chriskicker) [reply platform/authorized_api/apps/index.json]: both descriptions are for params (`gdprUserModeEnabled` and `privacyCookieExpirationPeriod`) introduced in 6.2

File: platform/authorized_api/apps/index.json [hunk 3]
```diff
@@ -41,6 +41,25 @@
...
+      "type": "string",
+      "description": "App type",
+      "example": "web",
+      "enum": [
+        "web",
+        "sharepoint"
+      ],
+      "default": "web"
+    },
+    "Pagination": {
```
Comment (RomanSek) [inline platform/authorized_api/apps/index.json]: Looks like it's not used in the file now.

**Pull request merged**